style.css

```
@@ -682,7 +629,7 @@ footer.showcase-footer {
 
 .index::after {
   content: "";
-  background: url(https://blackberry-py.nyc3.cdn.digitaloceanspaces.com/circuit_board_overlay.png) no-repeat center;
+  background: url(images/circuit_board_overlay.png) no-repeat center;
```

Stylelint (17.2.0)

[error] Expected quotes around "url" function argument (function-url-quotes)

(function-url-quotes)